### PR TITLE
Export top level lib files

### PR DESCRIPTION
**Pull request by blink1073**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -40,7 +40,9 @@
   "files": [
     "lib/*/*.js",
     "lib/*/*.d.ts",
-    "lib/*/*.css"
+    "lib/*/*.css",
+    "lib/index.js",
```
Comment (sccolbert) [inline package.json]: why not including glob js and dts?

Comment (blink1073) [inline package.json]: In this case the index is just a pass-through to the underlying libs, so there should only ever be the one file.

Comment (sccolbert) [inline package.json]: these are the files included with the npm package. You have to list everything that will be needed at runtime.

Comment (sccolbert) [inline package.json]: As in, why not just say `lib/*.js` `lib/*.d.ts`

**Comment by blink1073** [comment]
We can try it with this syntax and see if it works.

**Pull request merged**